netbout-web/src/main/java/com/netbout/rest/TkStart.java

```diff
@@ -67,13 +82,35 @@ public TkStart(final Base bse) {
 
     @Override
     public Response act(final Request req) throws IOException {
-        final Inbox inbox = new RqAlias(this.base, req).alias().inbox();
+        final Href href = new RqHref.Base(req).href();
+        final Iterator<String> token = href.param("token").iterator();
+        String tokenkey = "";
+        final Inbox inbox;
+        while (token.hasNext()) {
+            tokenkey = token.next();
+        }
+        if (tokenkey.isEmpty()) {
+            inbox = new RqAlias(this.base, req).alias().inbox();
+        } else if (this.tokens.containsKey(tokenkey)) {
+            final Object[] cache = this.tokens.get(tokenkey);
```

Comment:
@prahladyeri Let's call this `cached`, it better reflects what object it's referring to (the actual cache is `tokens`).